lib/button/client.rb

```diff
@@ -34,6 +35,7 @@ def initialize(api_key, config = {})
       @merchants = Merchants.new(api_key, config_with_defaults)
       @offers = Offers.new(api_key, config_with_defaults)
       @orders = Orders.new(api_key, config_with_defaults)
+      @tranasactions = Transactions.new(api_key, config_with_defaults)
```

Comment:
Looks like a misspelling, can you double check?

Reply:
Indeed! Thanks for the catch!